src/app/static/src/app/components/forms/FormControl.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <b-col :md="colWidth">
+        <label v-if="control.label">{{control.label}}</label>
```

Comment:
I wonder if there's some way to avoid having to put this v-if in for every type of control component. Using dynamic components somehow? https://alligator.io/vuejs/dynamic-components/
Could the control.type map exactly to the tag of the control component?
`<component is:="control.type">`?

Reply:
Much nicer, thanks for the prompt: https://github.com/mrc-ide/hint/pull/98/files#diff-88e4ce74e07a86408ea587fc3edef44eR4